libs/vm-modules/src/math/tensor.cpp

```diff
@@ -39,46 +40,70 @@ using SizeVector = ArrayType::SizeVector;
 VMTensor::VMTensor(VM *vm, TypeId type_id, std::vector<uint64_t> const &shape)
   : Object(vm, type_id)
   , tensor_(shape)
+  , estimator_(*this)
 {}
 
 VMTensor::VMTensor(VM *vm, TypeId type_id, ArrayType tensor)
   : Object(vm, type_id)
   , tensor_(std::move(tensor))
+  , estimator_(*this)
 {}
 
 VMTensor::VMTensor(VM *vm, TypeId type_id)
   : Object(vm, type_id)
+  , estimator_(*this)
 {}
 
 Ptr<VMTensor> VMTensor::Constructor(VM *vm, TypeId type_id, Ptr<Array<SizeType>> const &shape)
 {
   return Ptr<VMTensor>{new VMTensor(vm, type_id, shape->elements)};
 }
 
+/////////////////////////////////////////////////////////
+// Bind member function and their estimators to module //
+/////////////////////////////////////////////////////////
```

Comment:
Remove this comment - it adds nothing that the function prototype below doesn't already tell us.